kata-deploy/scripts/kata-deploy.sh

```diff
@@ -36,7 +38,11 @@ function get_container_runtime() {
 	if [ "$?" -ne 0 ]; then
                 die "invalid node name"
 	fi
-	echo "$runtime" | awk -F'[:]' '/Container Runtime Version/ {print $2}' | tr -d ' '
+	if echo "$runtime" | grep -qE 'Container Runtime Version.*containerd.*-k3s'; then
+		echo "containerd-k3s"
```

Comment:
should we rename this to just k3s or k3s-agent, so we can restart the service based on the name, like how it works w/ containerd/crio?